tests/eas/load_tracking.py

```diff
@@ -692,7 +692,39 @@ def _get_stable_window(self, df, task):
                              start + phase_duration_s)
         return window
 
-    def _get_util_mean(self, start, end, df, cpus):
+    def _get_idle_mean(self, tasks, cpus, phase):
+        """
+        Returns the mean idle % of all tasks running in each cpu for a given
+        phase
+        :param tasks: json description of the tasks
+        :type tasks: dict
+
+        :param cpus: list of CPUs for which the expected utilization needs to
+                     be computed
+        :type cpus: [int]
+
+        :param phase: name of the phase for which the expected utilization
+                      needs to be computed
+        :type phase: str
+        """
+        idle_mean = {}
+        for cpu in cpus:
+            tasks_count = 0
+            idle_pct = 0
+            for task in tasks.iteritems():
+                [phase_cpu] = self._get_cpus(task[1], phase)
+                if phase_cpu == cpu:
+                    idle_pct += 100 - self._get_duty_cycle(task[1], phase)
+                    tasks_count += 1
+
+            if tasks_count == 0:
```

Comment:
Duck-typing-friendly practice is to never compare to 0.
In that case, the if statement can be simplified into:
```
tasks_count = tasks_count or 1
```